bash fixes

susman opened the pull request:
add shebang
exit on errors
properly define current dir
replace awk by sed

0i0 commented:
setting up launcDeamond...
sed: 1: "s/user/leo/;s/pwd//User ...": bad flag in substitute command: 'U'
Liors-MacBook-Pro:geektoolio leo$

susman commented:
right.. moment

susman committed:
add shebang
exit on errors
properly define current dir
replace awk by sed
replace sed delimiters
